docs/installation.md

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_label: Installation
+title: Installation
+---
+
+First, make sure you have [Node.js](https://nodejs.org) version 16 or higher with NPM installed on your computer:
+
+```bash
+node --version
+npm --version
+```
+
+Install or upgrade Apify CLI by running:
+
+```bash
+npm -g install apify-cli
+```
+
+If you receive an `EACCES` error, you might need to run the command as root:
+
+```bash
+sudo npm -g install apify-cli
```

Comment:
What is the stance on the global install vs `npx apify-cli` here? Do we want to get rid of this here?

Reply:
i'd say the page should describe both approaches. in the end, `npx` installs to global, it is pretty much the same (but prefers local installs, which is nice). if you want to manually update what npx is using, you use the very same `npm -g i` command, right?